backend/public-graph/graph/resolver.go

```diff
@@ -803,67 +781,22 @@ func (r *Resolver) HandleErrorAndGroup(ctx context.Context, errorObj *model.Erro
 		errorObj.Event = strings.Repeat(errorObj.Event[:ERROR_EVENT_MAX_LENGTH], 1)
 	}
 
-	// stackTrace slice is set when we have a structured stacktrace input coming from ProcessPayload (frontend error)
-	// stackTraceString is set when we have a string input coming from ProcessBackendPayload (backend error)
-	// If there was no stackTraceString passed in, marshal it as a JSON string from stackTrace
-	if len(stackTrace) > 0 {
-		if stackTrace[0] != nil && stackTrace[0].Source != nil && (strings.Contains(*stackTrace[0].Source, "https://static.highlight.run/index.js") || strings.Contains(*stackTrace[0].Source, "https://static.highlight.io")) {
```

Comment:
Removed this re-routing. 
See #5324

I'm not sure if these will re-route to `project_id=1` but my thoughts are that we should:

* reach out to the workspace and see if we can get them to update their client config
* drop the error if we can't link it to a project